workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityGameObjectCommandHandlersGenerator.tt

```diff
@@ -108,13 +110,55 @@ namespace <#= qualifiedNamespace #>
                         return -1;
                     }
 
+                    Action<<#= receivedResponseType #>> wrappedCallback = response =>
+                    {
+                        if (this.IsValid() && callback != null)
+                        {
+                            callback(response);
+                        }
+                    };
+
                     var ecsCommandRequestSender = entityManager.GetComponentData<CommandSenders.<#= commandDetails.CommandName #>>(entity);
                     var request = <#= commandDetails.CommandName #>.CreateRequest(entityId, payload, timeoutMillis, allowShortCircuiting, callback);
                     ecsCommandRequestSender.RequestsToSend.Add(request);
                     return request.RequestId;
                 }
 
 <# } #>
+                long ICommandRequestSender.SendCommand<TCommand, TRequest, TResponse>(CommandTypeInformation<TCommand, TRequest, TResponse> commandTypeInformation, EntityId entityId, TRequest request,
```

Comment:
commandTypeInformation -> is that passed in so they don't need to explicitly define the generics when calling `SendCommand`?

Reply:
It also uniquely identifies the command. A (TRequest, TResponse) pair is not sufficient

Reply:
What he said